### PR TITLE
add italian streaming service

### PR DESCRIPTION
**Pull request by tiberio87**
Please add italian streaming service

### CODE REVIEW

**Comment by Audionut** [comment]
Thanks, but this doesn't look correct. It needs to follow the style of:
`'PSN': 'PSN', 'Playstation Network': 'PSN'`

ie:
shortname: shortname, longname: shortname

**Comment by tiberio87** [comment]
I changed it to the correct format and added another stream service

**Comment by Audionut** [comment]
https://en.wikipedia.org/wiki/Pirated_movie_release_types#Release_formats

```html
TIMV | TIMvision (formerly Cubovision)
```

Correct shortnames please.

**Comment by tiberio87** [comment]
ok, i ran tim vision

**Comment by Audionut** [comment]
And the others that were added?

File: src/prep.py [hunk 1]
```diff
@@ -3050,20 +3050,20 @@ def get_service(self, video=None, tag=None, audio=None, guess_title=None, get_se
             'Foxtel': 'FXTL', 'FYI': 'FYI', 'FYI Network': 'FYI', 'GC': 'GC', 'NHL GameCenter': 'GC', 'GLBL': 'GLBL',
             'Global': 'GLBL', 'GLOB': 'GLOB', 'GloboSat Play': 'GLOB', 'GO90': 'GO90', 'GagaOOLala': 'Gaga', 'HBO': 'HBO',
             'HBO Go': 'HBO', 'HGTV': 'HGTV', 'HIDI': 'HIDI', 'HIST': 'HIST', 'History': 'HIST', 'HLMK': 'HLMK', 'Hallmark': 'HLMK',
-            'HMAX': 'HMAX', 'HBO Max': 'HMAX', 'HS': 'HTSR', 'HTSR': 'HTSR', 'HSTR': 'Hotstar', 'HULU': 'HULU', 'Hulu': 'HULU', 'hoichoi': 'HoiChoi', 'ID': 'ID',
+            'HMAX': 'HMAX', 'HBO Max': 'HMAX', 'HS': 'HTSR', 'HTSR': 'HTSR', 'HSTR': 'Hotstar', 'HULU': 'HULU', 'Hulu': 'HULU', 'hoichoi': 'HoiChoi', 'ID': 'ID', 'INFINITY': 'INFINITY', 'Mediaset Infinity': 'INFINITY',
```
Comment (Audionut) [inline src/prep.py]: This has also been inserted between the shortname ID and the longname on the following line.

**Review by Audionut — CHANGES_REQUESTED** [review]

File: src/prep.py (hunk 1 above)
Comment (tiberio87) [reply src/prep.py]: OK, i've resolve error

**Comment by Audionut** [comment]
Pushed here https://github.com/Audionut/Upload-Assistant/commit/9c5e7847c10c95a977e5a7731363ec1967f0b14e

**Pull request closed without merging**